queue-client/src/main/java/tdl/participant/queue/events/VideoRecorderStartedEvent.java

```diff
@@ -0,0 +1,24 @@
+package tdl.participant.queue.events;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+import tdl.participant.queue.connector.QueueEvent;
+
+@Value
+@QueueEvent(name = "videoRecorderStarted", version = "0.2")
+public class VideoRecorderStartedEvent implements ParticipantEvent {
```

Comment:
Let's call it RecorderStartedEvent as the candidate can choose to disable video.